modules/lang-painless/src/main/resources/org/elasticsearch/painless/java.util.regex.txt

```diff
@@ -29,5 +29,31 @@ class Pattern -> java.util.regex.Pattern extends Object {
 }
 
 class Matcher -> java.util.regex.Matcher extends Object {
+  int end()
+  int end(int)
+  boolean find()
+  boolean find(int)
+  String group()
+  String group(int)
+  # String group(String)  How do I get this working?
```

Comment:
I'm fine with it because we have the aliases. I've since exposed this as namedGroup/1 and that is fine by me. If we end up keeping this forever (and it sounds like that is a good idea) then it'll always be a funny wart we explain away as being very useful for dynamic performance. Hell, we could just point people to this post to explain it.